agh_graphs/utils.py

```diff
@@ -0,0 +1,47 @@
+"""
+Utility module.
+"""
+import uuid
+
+from networkx import Graph
+
+
+def gen_name():
+    return str(uuid.uuid1())
+
+
+def centroid(a, b, c):
```

Comment:
I don't insist because `centroid` is quite meaningful but maybe one sentence of docs here? E.g. what are a, b and c.